docker/requirements.txt

```diff
@@ -1,6 +1,6 @@
 affine==2.3.1
-alabaster==0.7.12
-attrs==22.1.0
+appdirs==1.4.4
```

Comment:
Out of curiosity, why pin these reqs all the way down to the patch version? Why not just major.minor?

Reply:
Actually, there's been a change in the recent enhancement branch merge that simplified the Docker image - requirements.txt nows just a pip freeze of what is installed (maybe misnomer) for dependabot purposes.  The install was actually based on the much more sparse environment.yml.

Keen to here your thoughts on this approach, but there is less pinning in that file.